docs/reference/linstorcluster.md

```diff
@@ -0,0 +1,203 @@
+# `LinstorCluster`
+
+This resource controls the state of the LINSTOR cluster and integration with Kubernetes.
+In particular it controls:
+
+* LINSTOR Controller
+* LINSTOR CSI Driver
+* `LinstorSatellite`, configured via `LinstorSatelliteConfiguration` resources.
+
+### `.spec.nodeSelector`
+
+Selects on which nodes Piraeus Datastore should be deployed. Nodes that are
+excluded by the selector will not be able to run any workload using a Piraeus volume.
+
+If empty (the default), Piraeus will be deployed on all nodes in the cluster.
+
+#### Example
+
+This example restricts Piraeus Datastore to nodes matching `example.com/storage: "yes"`:
+
+```yaml
+apiVersion: piraeus.io/v1
+kind: LinstorCluster
+metadata:
+  name: linstorcluster
+spec:
+  nodeSelector:
+    example.com/storage: "yes"
+```
+
+### `.spec.repository`
+
+Sets the default image registry to use for all Piraeus images. The full image name is
+created by appending an image identifier and tag.
+
+If empty (the default), Piraeus will use `quay.io/piraeusdatastore`.
+
+The current list of default images is available [here](../../config/manager/default_images.yaml).
+
+#### Example
+
+This example pulls all Piraeus images from `registry.example.com/piraeus-mirror`
+instead of `quay.io/piraeusdatastore`.
+
+```yaml
+apiVersion: piraeus.io/v1
+kind: LinstorCluster
+metadata:
+  name: linstorcluster
+spec:
+  repository: registry.example.com/piraeus-mirror
+```
+
+### `.spec.properties`
+
+Sets the given properties on the LINSTOR Controller level, applying them to the whole Cluster.
+
+#### Example
+
+This example sets the port range used for DRBD volumes to `50000-60000`.
```

Comment:
Sounds like fun, using the conventionally dynamically allocated ports for DRBD :hand_over_mouth: